core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.spec.ts

```diff
@@ -371,6 +376,52 @@ describe('Admin dev mode activities tab', () => {
     );
   });
 
+  describe('.generateDummyTranslationOpportunities', () => {
+    it('should generate dummy explorations', waitForAsync(() => {
+      component.numDummyTranslationExpsToGenerate = 2;
+
+      spyOn(
+        adminBackendApiService,
+        'generateDummyTranslationOpportunitiesAsync'
+      ).and.returnValue(Promise.resolve());
+      spyOn(component.setStatusMessage, 'emit');
+      component.generateDummyTranslationOpportunities();
+
+      expect(component.setStatusMessage.emit).toHaveBeenCalledWith(
+        'Processing...'
+      );
+
+      fixture.whenStable().then(() => {
+        expect(component.setStatusMessage.emit).toHaveBeenCalledWith(
+          'Dummy translation opportunities (explorations) generated successfully.'
+        );
+      });
+    }));
+
+    it('should show error message when dummy translation opportunities are not generated', waitForAsync(() => {
+      component.numDummyExpsToPublish = 2;
```

Comment:
Can this test pass without this line as of now? If so, then I think it should be removed. I'm not seeing how setting this property relates to this specific outcome from the test scenario.

Reply:
Yeah, I removed it and the test passed without this line.